easybuild/easyconfigs/m/Molcas/Molcas-8.2-centos-mkl-par.eb

```diff
@@ -14,9 +14,10 @@ connected to the treatment of highly degenerate states."""
 
 toolchain = SYSTEM
 
-# download requires registration, see http://www.molcas.org/download.html
 sources = ['%(namelower)s%(version_major)s%(version_minor)s%(versionsuffix)s.tar.gz']
 checksums = ['54497a1b3e482802c05d0320c2e4fd57fb0f0c6f47b950517ae46e24d4da3b87']
+download_instructions = f"""{name} requires manual download from http://www.molcas.org/download.html
```

Comment:
use {homepage} and switch homepage to https://www.molcas.org

Reply:
2b72eac